### DISCUSSION TITLE
(IOS only) components outside of the ScrollView flicker when <Video/> loads

### DISCUSSION THREAD

**Discussion by taehun0933** [TheWidlarzGroup/react-native-video#4287] [Q&A]
### What are you trying to do?

(iOS only) Components outside of the ScrollView flicker when `<Video/>` loads.

While developing using react-native and react-native-video, I found a problem that only occurred on iOS, and I tried debugging it for quite a long time to solve it, but I can't tell it on my own, so I'm asking for help.

### What have you tried?

Debugging has confirmed so far that when there is a Video component inside ScrollView, the moment the video is loaded by the uri of this component, flickering occurs to the components outside ScrollView.

In addition to the simple flickering, there is also an intermittent break in the original layout (not the current code that only summarizes the problem logic for debugging). In this case, the ScrollView is raised 'as much as the area where the flickering has occurred', making it impossible to scroll.

A screen recording demonstrating the issue is attached: https://github.com/user-attachments/assets/e1b23f51-633d-4b93-a4ce-66a9c45825c1

Is there are some possible reasons?

### Platforms

iOS

**Comment by freeboub** [comment]
I think we can create a real ticket with that point ! please provide a reproducible sample to be able to investigate ! 
Thank you

**Comment by moskalakamil** [comment]
hey, closing as it should be issue- not disscussion. thanks!
